test/smock/MockBPool.sol

```diff
@@ -53,6 +53,14 @@ contract MockBPool is BPool, Test {
     return _totalWeight;
   }
 
+  function set__reenteringMutex(bytes32 __reenteringMutex) public {
+    _reenteringMutex = __reenteringMutex;
+  }
+
+  function call__reenteringMutex() public view returns (bytes32) {
+    return _reenteringMutex;
+  }
+
```

Comment:
what's this? 👀

Reply:
ahhh because the doc had to break transient storage, ok..

Reply:
autogenerated getters/setters for the storage `bytes32` introduced by the doc to walk around the limitation of `hevm` not supporting transient storage

Reply:
this is also what breaks the integration tests btw

Reply:
addressing this on #179